packages/story-editor/src/components/button/index.js

```diff
@@ -17,69 +17,15 @@
 /**
  * External dependencies
  */
-import { forwardRef } from 'react';
-import styled from 'styled-components';
+import { Button, BUTTON_VARIANTS } from '@web-stories-wp/design-system';
 
 /**
  * Internal dependencies
  */
-import { ArrowDown as ArrowDownIcon, More as MoreIcon } from '../../icons';
-
-const Base = styled.button.attrs(({ isDisabled }) => ({
-  disabled: isDisabled,
-}))`
-  border-width: 1px;
-  border-style: solid;
-  border-radius: 2px;
-  background: transparent;
-  display: block;
-  min-width: ${({ isIcon }) => (isIcon ? 'initial' : '63px')};
-  max-height: 30px;
-  padding: 0 10px;
-  cursor: pointer;
-
-  &:active {
-    outline: none;
-  }
-
-  svg {
-    width: 1em;
-  }
-
-  ${({ disabled }) =>
-    disabled &&
-    `
-		pointer-events: none;
-		opacity: .3;
-	`}
-`;
-
-const StyledButton = styled(Base)`
-  border: none;
-  padding: 0;
-  width: ${({ width }) => width}px;
-  height: ${({ height }) => height}px;
-  min-width: initial;
-  visibility: ${({ isHidden }) => (isHidden ? 'hidden' : 'visible')};
-  color: ${({ theme }) => theme.colors.standard.white};
-
-  svg {
-    width: ${({ width }) => width}px;
-    height: ${({ height }) => height}px;
-    display: block;
-  }
-`;
+import { ArrowDown as ArrowDownIcon } from '../../icons';
 
 export const ArrowDown = (props) => (
```

Comment:
Can ArrowDown also be removed here 🤔  

this ticket is all about cleaning up the redundancies of these now that we are using a design system. I did a brief check, and it seems to me that this button isn't getting used anywhere? Could you confirm and if that's the case this whole file could be removed.